lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1989,12 +1991,23 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
   {
     LLDB_SCOPED_TIMERF("%s (getStdlibModule)", m_description.c_str());
     const bool can_create = true;
+    std::unique_ptr<lldb_private::Progress> progress = std::make_unique<lldb_private::Progress>("Importing Swift standard library modules");
+    swift_ast_sp->m_ast_context_ap->SetPreModuleImportCallback(
+      [&progress](llvm::StringRef module_name, bool is_overlay) {
+        if (progress) {
+          progress->Increment(1, module_name.str());
```

Comment:
Seems like we're not using the `is_overlay` argument anymore. @adrian-prantl do you think this is relevant information to the user or should we just get rid of it in the callback ?

Reply:
What do you mean be "_we_ are not"? There is no overlay for the Swift stdlib module since it's implemented in Swift.

Reply:
This was not the best example: replied here were it makes more sense: https://github.com/apple/llvm-project/pull/7769#discussion_r1387283810